app/src/atoms/buttons/LargeButton.tsx

```diff
@@ -110,23 +113,36 @@ export function LargeButton(props: LargeButtonProps): JSX.Element {
       disabled={disabled}
       {...buttonProps}
     >
-      <StyledText
-        fontSize="2rem"
-        fontWeight={TYPOGRAPHY.fontWeightSemiBold}
-        lineHeight="2.625rem"
-      >
-        {buttonText}
-      </StyledText>
-      <Icon
-        name={iconName}
-        aria-label={`${iconName} icon`}
-        color={
-          disabled
-            ? COLORS.grey50
-            : LARGE_BUTTON_PROPS_BY_TYPE[buttonType].iconColor
-        }
-        size="5rem"
-      />
+      <Flex flexDirection={DIRECTION_COLUMN}>
+        <StyledText
+          fontSize="2rem"
```

Comment:
nit: can you use the constant
```suggestion
          fontSize={TYPOGRAPHY.fontSize32}
```